.github/workflows/deploy.yml

```diff
@@ -155,3 +155,26 @@ jobs:
         if: ${{ env.SEND_SLACK_MSG == 'true' }}
         run: |
           curl -X POST -H 'Content-type: application/json' --data '{"text":"${{ env.DOCS_SLACK_MSG }}"}' ${{ secrets.SLACK_WEBHOOK }}
+  image:
+    runs-on: ubuntu-22.04
+    if: github.ref == 'refs/heads/published'
```

Comment:
We want to build only on for published ref?

Reply:
Changed it to build on pr/main, and `push: ${{ github.event_name != 'pull_request' }}`